.buildkite/pipeline_pr.py

```diff
@@ -110,11 +110,14 @@
 steps = [step_style]
 changed_files = get_changed_files("main")
 
-# run sanity build of devtool if Dockerfile is changed
-if any(x.name == "Dockerfile" for x in changed_files):
+# run sanity build of devtool if Dockerfile or devtool are changed
+if any(x.name in "Dockerfile" for x in changed_files):
```

Comment:
It looks missing "devtool" in the list. It runs if "Dockerfile" or "devtool" are changed, doesn't it?

Reply:
ooops, yeah, originally I changed this `if` because I thought it did the `make_release` test run, and apparently I messed up reverting it. Let me fix

Reply:
fixed